WebRoot/css/zm.css

```diff
@@ -2176,7 +2168,42 @@ div.CompDragArea * {
 }
 
 .ZmTaskEditView {
-	@PageBg@
+	@TaskEditView@
+}
+
+.ZmTaskEditView .ZmTaskEditHeader {
+	@TaskEditHeader@
+}
+
+.ZmTaskEditView .ZmTaskEditContent {
+	@TaskEditContent@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable {
+	@TaskEditHeaderTable@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable > tbody > tr > td {
+	@TaskEditHeaderTableCell@
+}
+
+.ZmTaskEditHeaderTable .ZmFieldsetContent td,
+.ZmTaskEditHeaderTable .ZmFieldsetContent div,
+.ZmTaskEditHeaderTable .ZmFieldsetContent label {
+	@TaskEditViewFontSize@
```

Comment:
Actually this should be the case. But we can't do that, because we have CSS declarations somewhere that targets each element explicitly to set font size. So to override those rules, we have to declare CSS like this.

Regarding container, we already have container for `FieldSetContent`

Let me know if you have any questions.